src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -89,7 +89,11 @@ def _complex_params(command_group):  # pylint: disable=too-many-statements
             c.argument('assign_identity', options_list=['--assign-identity'], help='Generate and assign an Azure Active Directory Identity for this server for use with key management services like Azure KeyVault.')
 
             c.argument('location', arg_type=get_location_type(self.cli_ctx))
-            c.argument('version', help='Server major version.')
+            if command_group == 'postgres':
+                c.argument('version', default=11,
```

Comment:
The version seems to expect a `str` value but you set a `int` value as default

Reply:
done